swag_runbook.md

```diff
@@ -4,50 +4,107 @@ Use this template to write up a runbook for your area of concern.
+* Sponsor Lanyards
+  - Select vendor, designs for lanyard
 
 ## Important Dates
 
 What dates are important for this role? The following headings and details are examples.
 
 ### In the beginning
 
-* Come up with good stage jokes.
+* What is the theme for the conference.  This will influence the logo
 
 ### 3 Months Out
 
-* Find vendors for your stage costume.
+* Start the logo process.
+* Identify candidates to create the logo
+* Find vendors for shirts & gifts
+* Find and work with artists (artist search post and contract)
+* Review budget & requirements (# of attendees)
 
 ### 2 Months Out
 
-* Try on the costume.
+* Meet to finalize the following
+  * Artist(s) and the designs **need this to move forward with the rest**
+  * Speaker gifts (Identify production time)
+  * Start engage vendors on pricing based on expected attendance.
+    * Shirts
+      * Day 1 and Day 2
+      * Organizers
+      * Volunteers
+    * Speaker gifts
+    * Start designing the badge. Usually the logo is involved and a double sided badge with name on both sides is preferable.  Typically the vendor will help with the desogn process.
+    * Start the lanyard process based on expected number of attendees.  The lanyards must at a minimum have the sponsors logo and it is nice to have the DevOpsDaysAustin logo.  A lot of time was spent back and forth about getting the DevOpsDaysAustin logo on the lanyard.
+* Review current budget
 
 ### 1 Month Out
 
-* Sell out the sponsorships.
+* Order organizer shirts
+* Order lanyards 
+* Order badges (with some blank cards)
+* Get t-shirt samples with colors and various sizes
+* Get gift samples
+* Meet to finalize gifts & shirts
+* Order gifts
+
+### 3 Weeks out
+
+* Start printing shirts
+* Really make sure you have a handle on badges.  This was very stressful in that we were not sure they would make it in time.
+
+### 2 Weeks Before
+* Order t-shirts with current attendees sizes with buffer for limit
+* Order shirts with current volunteers sizes with buffer
+* Order shirts for organizers
 
 ### The Week Before
 
-* Panic.
+* Pick up shirts, gifts, lanyards, and badges
+* Print final Speakers List - work with the speakers lead
+* Print volunteers/org list - work with volunteers lead
 
 ### The Day Before
 
-* Set up.
-* Party.
+* Fix up the badges if they need to be in plastic sleeves, etc
+* Handout organizer shirts
+* Fold T Shirts
+* Hand out speaker gifts to speaker dinner.
 
 ### The Day Of
 
-* Get on stage.
+* Be Excellent
+* Set up
+  * ensure there's a printing machine or marker for the blank badges
+  * setup shirt station
+  * setup badge station
+* Ensure volunteers know where swags are located and organized
 
-## Reminders or Common Mistakes
 
-What do you always try to remember when you come back to this every year? Do you often find yourself scrambling for vendors? Are you always trying to remember where you left your towel? Leave notes for the next person so we can improve :)
+## Reminders or Common Mistakes
```

Comment:
Maybe add a bit about holding some swag back so that we have some to give next year, people love vintage swag :P